src/NuGet.Core/NuGet.ProjectModel/IncludeExcludeFiles.cs

```diff
@@ -86,5 +86,15 @@ public bool Equals(IncludeExcludeFiles other)
                    IncludeFiles.SequenceEqualWithNullCheck(IncludeFiles) &&
                    ExcludeFiles.SequenceEqualWithNullCheck(ExcludeFiles);
         }
+
+        public IncludeExcludeFiles Clone()
+        {
+            var clonedObject = new IncludeExcludeFiles();
```

Comment:
I don't think this is even used, maybe we could drop support for cloning some of these properties?

Reply:
That could be the thing. 
I found a bunch of things that I don't see being used anymore in productive code, but actually being read in read/write.

Reply:
🔪  they are all remnants of project.json, and things from DNU before this was ported to NuGet. Now is a good time to actually deprecate those.

Reply:
+1, it will also reduce the burden of packageSpec read wrtite which ultimately impact assets file read write as well.